stratum/hal/lib/common/p4_service.cc

```diff
@@ -384,6 +384,18 @@ ::grpc::Status P4Service::SetForwardingPipelineConfig(
                      node_id, "."));
   }
 
+#ifdef P4OVS_CHANGES
```

Comment:
This is supported on Tofino. We should make the alteration in the tdi_node code or lower perhaps in the per target code. SetForwardingPipeline is a commonly used API in P4Runtime and expected to be supported on targets supporting P4Runtime.